src/main/java/com/github/princesslana/eriscasper/rest/Route.java

```diff
@@ -31,7 +31,9 @@
     GET("GET", Content.QUERY_STRING),
     PATCH("PATCH", Content.BODY),
     POST("POST", Content.BODY),
-    PUT("PUT", Content.BODY);
+    QUERY_POST("POST", Content.QUERY_STRING),
+    PUT("PUT", Content.BODY),
+    QUERY_PUT("PUT", Content.QUERY_STRING);
```

Comment:
Sending a request body along with these requests has not been addressed?

Reply:
yes it has, it doesn't need body content, check comments on your previous comment